Angular InstantSearch/geo-search/package.json

```diff
@@ -33,7 +33,6 @@
     "@angular/cli": "7.3.8",
     "@angular/compiler-cli": "6.1.10",
     "@angular/language-service": "6.1.10",
-    "@types/algoliasearch": "3.27.8",
```

Comment:
why was this removed?

Reply:
Because we import `algoliasearch/lite` which does not have the types. So it's unused.

Reply:
lite does have the types though: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/algoliasearch/lite/index.d.ts

Reply:
Oh interesting! So I have this error in the latest version:

> ERROR in node_modules/@types/algoliasearch/lite/index.d.ts(14,43): error TS2304: Cannot find name 'Exclude'.

This is the error raised in algolia/algoliasearch-client-javascript#753.

We can move forward with this PR and add them when fixed.

Reply:
As discussed, this is because the example uses 2.7, instead of a more recent version (2.8+) which comes with Exclude, which is written in the preamble of the definitions